src/korean.js

```diff
@@ -104,7 +104,7 @@ class Korean {
 
   doPast(word) {
     let presentWord = doPresent(word);
-    // stuff for past tense
+    return presentWord.concat('ㅆ어')
```

Comment:
Hi @thisisrai, thank you for contributing!

Unfortunately, I don't think you're code will work.
You have to use the breakdown and combineSymbols functions to to combine 2 or 3 jamo into one character. Here is some more information : 
http://www.programminginkorean.com/programming/hangul-in-unicode/composing-syllables-in-unicode/
Could you also add some testcases on the korean_tests.js file for the doPast function? We take the habit to write our own tests with each function we write.
This way you can then test you're code by running npm run test in your terminal.